lahja/asyncio/endpoint.py

```diff
@@ -277,7 +279,7 @@ class AsyncioEndpoint(BaseEndpoint):
     _receiving_queue: "asyncio.Queue[Tuple[Union[bytes, BaseEvent], Optional[BroadcastConfig]]]"
     _receiving_loop_running: asyncio.Event
 
-    _futures: Dict[Optional[str], "asyncio.Future[BaseEvent]"]
```

Comment:
I'm surprised it was even possible for the key of a dictionary to be optional :smile:

Reply:
Just mean `None` as a key which is possible, but you can only ever have one.

Reply:
This is one of the few occasions where `Union[str, None]` (which is equivalent to `Optional[str]`) may *feel* a little more suitable :sweat_smile: